python-squid/src/test/java/org/sonar/python/semantic/SymbolTableBuilderTreeTest.java

```diff
@@ -49,10 +52,18 @@
   @BeforeClass
   public static void init() {
     PythonVisitorContext context = TestPythonVisitorRunner.createContext(new File("src/test/resources/semantic/symbols2.py"));
-    FileInput fileInput = context.rootTree();
+    fileInput = context.rootTree();
     fileInput.accept(new TestVisitor());
   }
 
+  @Test
+  public void module_variable() {
+    Set<Symbol> moduleSymbols = fileInput.moduleFields();
+    List<String> names = Arrays.asList("global_x", "global_var");
+    assertThat(moduleSymbols.size()).isEqualTo(2);
+    moduleSymbols.forEach(s -> assertThat(names).contains(s.name()));
```

Comment:
`assertThat(moduleSymbols).extracting(name).containsExactly("global_x", "global_var");`

Reply:
Doesn't this assert on the order as well? which can't be guaranteed for a Set?